### PR TITLE
APB-6123 [CS] existing team members page

### PR DESCRIPTION
**Pull request by CarolineS-QA**

### CODE REVIEW

**Review by geoffreywatson — DISMISSED** [review]

File: app/controllers/ManageGroupController.scala [hunk 1]
```diff
@@ -236,44 +236,28 @@ class ManageGroupController @Inject()(
...
-    //                      routes.ManageGroupController.showManageGroupTeamMembers(groupId).url
-    //                    )).toFuture
-    //                }
-    //            }
-    //          })
-    //        )
-    //      }
-    //    }
+  def showExistingGroupTeamMembers(groupId: String): Action[AnyContent] = Action.async { implicit request =>
+    isAuthorisedAgent { arn =>
```
Comment (max-hmrc) [inline app/controllers/ManageGroupController.scala:240]: You can replace lines 240 to 243 with
 withGroupForAuthorisedOptedAgent(groupId, (group: AccessGroup) => {

**Review by max-hmrc — CHANGES_REQUESTED** [review]

File: app/controllers/ManageGroupController.scala (hunk 1 above)
Comment (CarolineS-QA) [reply app/controllers/ManageGroupController.scala:240]: if I do that the arn isn't accessible on line 249
` groupService.getTeamMembersFromGroup(arn)(Some(teamMembers)).flatMap(groupMembers =>`

Comment (max-hmrc) [reply app/controllers/ManageGroupController.scala:240]: I think arn is in the AccessGroup itself

Comment (max-hmrc) [reply app/controllers/ManageGroupController.scala:240]: group.arn

Comment (CarolineS-QA) [reply app/controllers/ManageGroupController.scala]: ahh okay, thanks!

**Review by max-hmrc — APPROVED** [review]

**Pull request merged**